lms/views/admin/application_instance.py

```diff
@@ -165,17 +139,7 @@ def upgrade_instance(self, lti_registration, consumer_key):
 
         # Set the LTI1.3 values
         application_instance.lti_registration = lti_registration
-        application_instance.deployment_id = self.request.params[
-            "deployment_id"
-        ].strip()
-
-        # And upgrade any other values present in the request
-        if email := self.request.params["email"].strip():
```

Comment:
We focus on upgrading. If you want to change any of this value you can do it directly on the application instance detail view.